R/Spectra.R

```diff
@@ -1209,7 +1250,33 @@ setMethod("clean", "Spectra",
               object
           })
 
-## compareSpectra
+#' @rdname Spectra
+#'
+#' @exportMethod compareSpectra
+#'
+#' @export ppm closest
+setMethod("compareSpectra", signature(x = "Spectra", y = "Spectra"),
+          function(x, y, FUN = cor, tolerance = 0, ppm = 20, ...,
+                   SIMPLIFY = TRUE) {
+              mat <- .compare_spectra(x, y, FUN = FUN, tolerance = tolerance,
+                                      ppm = ppm, ...)
+              if (SIMPLIFY && (length(x) == 1 | length(y) == 1))
```

Comment:
Doesn't really matter here but would be possible:
```suggestion
              if (SIMPLIFY && (length(x) == 1 || length(y) == 1))
```